frontend/app/components/checkin-dialog.js

```diff
@@ -2,8 +2,7 @@ import Ember from 'ember';
 import StepControl from 'flaredown/mixins/step-control';
 
 export default Ember.Component.extend(StepControl, {
-
-  classNameBindings: ['isSummary:checkin-summary:flaredown-white-box'],
+  tagName: '',
```

Comment:
You are right about `checkin-summary`, I don't find any reason to do so. It is some kind of leftover.

The `isSummary` was checked 4 times in conditional statements (3 on template and 1 in class name binding).
Now it is checked only once.